src/main/webapp/styles/itinerary-overview.css

```diff
@@ -0,0 +1,10 @@
+#map{
+    height: 100%;
+}
+
+html,
+body{
```

Comment:
It's better not to use selectors on all of the html, but since this is a demo it's okay.

Reply:
What would you use in its place?

Reply:
Maybe you can make a container div that has 100% height and width. I think we don't like to select entire HTML tag types because it can cause unexpected behavior when maintaining a certain page in the future.

Reply:
Ok I will change some stuff in the CSS file on the next PR so I'll handle it then.